workflows/ascc_genomic.nf

```diff
@@ -113,6 +115,86 @@ workflow ASCC_GENOMIC {
     }
 
 
+    //
+    // SUBWORKFLOW: RUN SOURMASH TO GET TAXONOMIC INFORMATION ABOUT
+    //             SCAFFOLDS IN ASSEMBLY
+
+
+    //
+    // LOGIC: PARSE SOURMASH DATABASE CONFIGURATION
+    //
+    def sourmashConfig = new SourmashDatabaseConfig()
+    def sourmash_databases = sourmashConfig.parseDatabaseConfig(params)
+
+    // Validate databases if Sourmash is enabled
+    if (params.run_sourmash == "both" || params.run_sourmash == "genomic") {
+        if (sourmash_databases.size() > 0) {
+            def validation = sourmashConfig.validateDatabases(sourmash_databases)
+
+            // Log warnings
+            validation.warnings.each { warning ->
+                log.warn "[ASCC Sourmash] ${warning}"
+            }
+
+            // Log summary if valid
+            if (validation.valid) {
+                log.info sourmashConfig.getDatabaseSummary(sourmash_databases)
+            } else {
+                log.error "[ASCC Sourmash] Database validation failed. Sourmash will be skipped."
```

Comment:
This should be a single if else, 
if valid then run sourmash, else pipeline error